### PR TITLE
napari-aicsimageio v0.7.0

### PR DESCRIPTION
**Pull request by tlambert03**
closes #6 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by tlambert03** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,18 +20,24 @@ requirements:
...
+    - aicsimageio >=4.6.3
     - fsspec
-    - napari >=0.4.11
-    - napari-plugin-engine >=0.1.4
+    - napari >=0.4.13
+    - psutil >=5.7.0
     - python >=3.8
+    # plugins
+    - bioformats_jar >=2020.5.27
+    - imageio >=2.9.0
```
Comment (tlambert03) [inline recipe/meta.yaml]: @evamaxfield ... how important was that <2.11.0 pin?  that will very likely cause problems for napari

Comment (tlambert03) [reply recipe/meta.yaml]: (note that napari has `!=2.11.0`)

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Home: https://github.com/AllenCellModeling/napari-aicsimageio
 
 Package license: BSD-3-Clause
```
Comment (evamaxfield) [inline README.md:6]: This package license is out of date if you have a second to update it!

**Review by evamaxfield — APPROVED** [review]
Minor ask of updating license but looks good to me!

File: recipe/meta.yaml (hunk 1 above)
Comment (evamaxfield) [reply recipe/meta.yaml]: Uhhhhhh somewhat important. Maybe just remove this then? The associated bug is here: https://github.com/AllenCellModeling/aicsimageio/issues/361

Haven't gotten around to updating usage yet.

Comment (tlambert03) [reply recipe/meta.yaml]: have you tried `2.12.0`?  that 2.11 version screwed us up too, but everything since has been fine

Comment (tlambert03) [reply recipe/meta.yaml]: (but will remove for now)

Comment (evamaxfield) [reply recipe/meta.yaml]: I haven't. I can try later today. ~4 hours from now ish.

Comment (evamaxfield) [reply recipe/meta.yaml]: Removing imageio for now works for me!

**Comment by tlambert03** [comment]
ok, should be good to go assuming tests pass

File: recipe/meta.yaml [hunk 3]
```diff
@@ -20,18 +20,23 @@ requirements:
...
+    - aicsimageio >=4.6.3
     - fsspec
-    - napari >=0.4.11
-    - napari-plugin-engine >=0.1.4
+    - napari >=0.4.13
+    - psutil >=5.7.0
     - python >=3.8
+    # plugins
+    - bioformats_jar >=2020.5.27
+    - imageio-ffmeg >=0.4.5
```
Comment (evamaxfield) [inline recipe/meta.yaml]:
```suggestion
    - imageio-ffmpeg >=0.4.5
```

Comment (evamaxfield) [reply recipe/meta.yaml]: (or remove this line entirely)

Comment (tlambert03) [reply recipe/meta.yaml]: 😂   woops.  removed

**Comment by tlambert03** [comment]
i removed `pip check` here... it's was failing because aicslibczi isn't on conda-forge, but we don't need it here.  it's also failing to detect the version of resource-backed-dask-array... that one I can't figure out.  all the pypi metadata is correct, and when I install from pypi, it gets the dist info version correct.  but for some reason if the file on conda forge https://anaconda.org/conda-forge/resource_backed_dask_array/files has the wrong version in the dist-info inside of the tar gz.  I'd need @jaimergp  to tell me what's going on with that one

**Pull request merged**